Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -49,6 +49,7 @@ class WebViewController: UIViewController, WebController {
         case focusTrackingProtection
         case focusTrackingProtectionPostLoad
         case findInPageHandler
+        case fullScreen
 
         static var allValues: [ScriptHandlers] { return [.focusTrackingProtection, .focusTrackingProtectionPostLoad, .findInPageHandler] }
```

Comment:
Does `fullScreen` need to be in this array?

Reply:
I thought it's needed for `browserView` configuration, like we use for `FindInPage`.